docs/spelling_wordlist.txt

```diff
@@ -156,3 +156,15 @@ doesn
 structuralizing
 pseudocode
 txt
+str
+bool
+FDict
+arg
+dep
+coreferent
+ConstituentNode
+ag
+umls
+cui
+FList
+qid
```

Comment:
Most of these words here are variables/programmatic terms, they are not supposed to be in the wordlist. You should wrap those with backtip, like this: `str`